### ISSUE TITLE
Trivial ansible-remote script does not work with AWS builder

### ISSUE THREAD

**Issue by pa-jama** [hashicorp/packer#5278]
My playbook works fine if I manually run it.
However packer tries to run it on 127.0.0.1 and it can't handshake?

Packer v1.0.4

[Full Log](https://gist.github.com/pa-jama/f7d5670c4782f7e3c197b0a09957e7da)

```
{
  "builders": [{
    "type": "amazon-ebs",
    "profile": "{{user `aws_profile`}}",
    "source_ami": "ami-e2021d81",
    "instance_type": "t2.micro",
    "ssh_username": "ubuntu",
    "ami_name": "appserver {{timestamp}}"
  }],
  "provisioners": 
  [{
    "type": "ansible",
    "user": "ubuntu",
    "playbook_file": "playbooks/all.yml",
    "groups": ["ubuntu16"],
    "extra_arguments": ["-vvvv"]
  }]
}

```
```
- hosts: ubuntu16
  become: true
  become_method: sudo
  become_user: root
  roles:
    - sysupdate
    - bootstrap/ubuntu
    - users
    - install/haproxy
    - utils

```

**Comment by mwhooker** [comment]
only advice I can offer is make sure you understand how the ansible remote provisioner works

> The ansible Packer provisioner runs Ansible playbooks. It dynamically creates an Ansible inventory file configured to use SSH, runs an SSH server, executes ansible-playbook, and marshals Ansible plays through the SSH server to the machine being provisioned by Packer. Note, this means that any remote_user defined in tasks will be ignored. Packer will always connect with the user given in the json config.

https://www.packer.io/docs/provisioners/ansible.html

and that you should make sure the host you run packer from is reachable over ssh

**Comment by pa-jama** [comment]
I've just resolved this. In my opinion, it's a bug though.
It reports a connection error but there is definitely not a connection error.

The problem is the interpreter needs to be ansible_interpreter=/usr/bin/python3 for 16.04
I fixed this by creating group_vars for ubuntu16 and setting the above variable.

Is there a way to set this directly in the json file? IE put the ansible_python_interpreter directly into the generated inventory file?
